integration_tests/18_block_runners.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Test for https://github.com/testground/testground/issues/1236
+my_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# block the daemon from starting automatically
+AUTO_START=1
```

Comment:
Usually, `0` is false-ish and `1` is truth-ish,
So when you write `AUTO_START=1` I will assume you mean "please auto start".

Let's use `SKIP_AUTO_START`.